src/Uno.UI/UI/Xaml/Window.Android.cs

```diff
@@ -136,9 +136,13 @@ private double GetLogicalNavigationBarHeightExcluded()
 		{
 			// The real metrics excluded the NavigationBar only if it is plain.
 			// We want to substract it if it is translucent. Otherwise, it will be like we substract it twice.
-			return NavigationBarHelper.IsNavigationBarVisible && NavigationBarHelper.IsNavigationBarTranslucent
-				? NavigationBarHelper.LogicalNavigationBarHeight
+			return ContextHelper.Current is Activity activity && ShouldBeExcluded()
+				? ViewHelper.PhysicalToLogicalPixels(LayoutProvider.Instance.NavigationBarLayout.Height())
 				: 0;
+
+			bool ShouldBeExcluded() => false
```

Comment:
false || ? Did you forget to remove this ?

Reply:
Intended to align the `or` conditions below, otherwise we end up with something hard to read, like:
```
var predicate = 
    condition1
    || condition2;
```

Reply:
If readers are wondering why there's a pointless `false`, it's still hard to read.